src/event_loop.rs

```diff
@@ -374,13 +374,26 @@ impl<T> EventLoop<T>
                         CHANNEL => if !self.channel_event(&mut state) {
                             break 'event_loop;
                         },
+
+                        #[cfg(unix)]
+                        token if token == self.pty.signal_token() => {
+                            let should_exit = self.pty.process_signals();
+                            if should_exit {
```

Comment:
Returning a `bool` here sucks, but I'm not sure what would be better.

Reply:
I'd either rename the method to something that clearly return a bool, or maybe return an enum with the type of signal that was processed? Or just an action that should be taken maybe.

Reply:
I'll try an `enum`, see how it feels.